web/wp-content/themes/dctx-v2/template-scaffolding.php

```diff
@@ -0,0 +1,20 @@
+<?php
```

Comment:
:no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected phpcs-scan-dudczz.php, but found phpcs-scan-DuDczz.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).